app/models/heart_rate_data.rb

```diff
@@ -0,0 +1,30 @@
+class HeartRateData < ApplicationRecord
+
+  belongs_to :user, dependent: :destroy
+
+  def update_heart_rate_data!
+    update!(data: heart_rate_data_for_the_day)
+  end
+
+  private
+
+  def heart_rate_data_for_the_day
```

Comment:
yeah - I'm fine with that. 'today' is convenient but the `YYYY-mm-dd` pattern is more consistent and easier to grok IMHO